engine/table/src/main/java/io/deephaven/engine/table/impl/hierarchical/BaseNodeOperationsRecorder.java

```diff
@@ -32,27 +32,33 @@
  */
 abstract class BaseNodeOperationsRecorder<TYPE> {
 
-    final TableDefinition definition;
+    private final TableDefinition initialDefinition;
 
     private final Collection<? extends SelectColumn> recordedFormats;
     private final Collection<SortColumn> recordedSorts;
     private final Collection<? extends SelectColumn> recordedAbsoluteViews;
+    private final Collection<? extends SelectColumn> recordedUpdateViews;
 
     volatile TableDefinition resultDefinition;
 
     BaseNodeOperationsRecorder(
             @NotNull final TableDefinition definition,
             @NotNull final Collection<? extends SelectColumn> recordedFormats,
             @NotNull final Collection<SortColumn> recordedSorts,
-            @NotNull final Collection<? extends SelectColumn> recordedAbsoluteViews) {
-        this.definition = definition;
+            @NotNull final Collection<? extends SelectColumn> recordedAbsoluteViews,
+            @NotNull final Collection<? extends SelectColumn> recordedUpdateViews) {
```

Comment:
We don't record the filters in the base, because they don't apply to the rollup in the same way that they do to trees; I think we could very likely say the same about updateViews and move them to the RollupOperationsRecorder only.